pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Allows to connect to the cloud powered by Pasqal to run sequences."""
+from typing import Dict, List, Mapping, Optional, Union
```

Comment:
As a general remark, the usage of `Dict` and `List` has been deprecated in favour of the built-in `dict` and `list`, which you can use if you import `annotations` from `__future__`. Afaik, the only case where you still need this is when using `cast`.